packages/fxa-settings/src/lib/sensitive-data-client.ts

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export class SensitiveDataClient {
+  private sensitiveData: { [key: string]: string } = {};
```

Comment:
The value is not always string though?  That's not the case for the usage in this PR.

Reply:
I'll fix this, this was when I was encrypting JSON value of data.